x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -255,7 +255,7 @@ export const allowedExperimentalValues = Object.freeze({
   /**
    *  Enables Security AI Assistant's Flyout mode
    */
-  aiAssistantFlyoutMode: false,
+  aiAssistantFlyoutMode: true,
```

Comment:
🙃

Reply:
while it's exciting to see this enabled, it feels like this PR isn't the place? @patrykkopycinski

Reply:
great catch! ❤️